scripts/runtests.sh

```diff
@@ -60,7 +60,7 @@ cd $BUGZILLA_ROOT
 ./checksetup.pl qa/config/checksetup_answers.txt
 
 echo -e "\n== Generating bmo data"
-generate_bmo_data.pl
+perl generate_bmo_data.pl
```

Comment:
It would be:

perl scripts/generate_bmo_data.pl

but works for me otherwise.